app.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+
+const app = express();
+
+app.get("/",(request, response) =>{
+    console.log("GET request received to route: /")
+    response.send("Welcome 99 Pokemon")
+} )
+
+app.get("/jumping/joyous/jellybean",(request,response) => {
+    response.send("Congratulations on starting a new project called jumping-joyous-jellybean!");
+})
+
+app.get("/bugs",(request,response) => {
+    response.send(`99 little bugs in the code
+    <a href = /bugs/101> pull one down, patch it around</a>  
+    `)
+})
+
+app.get("/bugs/101",(request,response) => {
```

Comment:
this path and the path below are very similar, however, the one on line 25 is dynamic.